packages/account/src/Configs/personal-details-config.ts

```diff
@@ -190,6 +190,24 @@ export const personal_details_config = ({
             supported_in: ['svg'],
             rules: [],
         },
+        crs_confirmation: {
+            default_value: false,
+            supported_in: ['svg'],
```

Comment:
is this applicable only for CR clients?

Reply:
Yes @amina-deriv this is only in CR account creation within the `Additional Information` section as we already have this confirmation for MF.